java/src/jmri/jmrit/conditional/ConditionalCopyFrame.java

```diff
@@ -482,20 +446,6 @@ public String getColumnName(int col) {
             return "";
         }
 
-        public int getPreferredWidth(int col) {
-            switch (col) {
-                case ROWNUM_COLUMN:
-                    return 10;
-                case NAME_COLUMN:
-                    return 200;
-                case DESCRIPTION_COLUMN:
-                    return 600;
-                default:
-                    break;
-            }
-            return 10;
-        }
```

Comment:
Why is this method removed? Is the same method in the parent class?
(I'm on the phone now so I have difficult to check).

Reply:
Thanks for double checking this.
This PR moves the containing class ( VariableCopyTableModel  ) from package private to private, improving encapsulation.
These methods then got flagged within Spotbugs as unused.
There is no parent method to override in CopyTableModel or AbstractTableModel.
I thought it was safer to remove the methods rather than to potentially mess up the table.

Reply:
`TableModel` doesn't have a `getPreferredWidth` method to override.  Note there's no `@Override` on this. This mthod is probably an oversight or copy&paste error from some `JTable` implementation in the past.  I think SpotBugs has it right that it's not being used.

Reply:
> I thought it was safer to remove the methods rather than to potentially mess up the table.

I agree.